definition-map.js

```diff
@@ -177,7 +177,7 @@ const MUTATIONS = {
         metadescription: "Creates a comment.",
         args: {
             commentableId: "The ID of the commentable entity.",
-            commentableType: "The type of the commentable entity.",
+            commentableType: "The type of comment to create, based on the comment's location, e.g. discussion board, assignment, widget thread etc.",
```

Comment:
This wouldn't really be the comment's type - it's the type of the comment's parent thread.